heidelberg-university-faculty-of-medicine.csl

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never">
+  <!-- Polyglot; used both with German and English -->
+  <info>
+    <title>Heidelberg University - Faculty of Medicine (Universität Heidelberg - Medizinische Fakultät Heidelberg)</title>
+    <id>http://www.zotero.org/styles/heidelberg-university-faculty-of-medicine</id>
+    <link href="http://www.zotero.org/styles/heidelberg-university-faculty-of-medicine" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-european-public-policy" rel="template"/>
+    <link href="http://www.medizinische-fakultaet-hd.uni-heidelberg.de/fileadmin/Dekanat/Abfassungsrichtlinien_Sept_2015.pdf" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="medicine"/>
+    <summary>Style for dissertations at the Faculty of Medicine at the University of Heidelberg.</summary>
+    <updated>2017-02-19T19:17:48+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
```

Comment:
@damnation333, if you leave out the "default-locale", I would change this to `<locale xml:lang="en">`, otherwise people who use it with "en-US" won't see the effects of this locale block.